README.md

```diff
@@ -8,14 +8,55 @@ description: "A collection of labs demonstrating how to build Python application
+# Azure Python Labs at PyCon India
 
+- [Azure Python Labs at PyCon India](#azure-python-labs-at-pycon-india)
+    - [:tada::snake: Welcome to out Python Virtual Labs for PyCon India Online 2020 :tada::snake:](#-welcome-to-out-python-virtual-labs-for-pycon-india-online-2020-)
+    - [:thinking: How to participate?](#-how-to-participate)
+  - [:microscope: Virtual labs](#-virtual-labs)
+    - [:heart::zap: Sentiment Analysis with Python Azure Functions](#️️-sentiment-analysis-with-python-azure-functions)
+    - [:sunny: Hello World in Visual Studio Codespaces](#️-hello-world-in-visual-studio-codespaces)
+    - [:page_facing_up: Document recognition with Python Azure Form Recognizer](#-document-recognition-with-python-azure-form-recognizer)
+    - [:mag: Explore Azure Database for PostgreSQL with Python](#-explore-azure-database-for-postgresql-with-python)
+    - [:100: Number Facts with Python Web Apps](#-number-facts-with-python-web-apps)
+    - [:whale: Developing a Django + PostgreSQL application in a Dev Container](#-developing-a-django--postgresql-application-in-a-dev-container)
+    - [:bug: Debugging a Flask App with WSL in VS Code](#-debugging-a-flask-app-with-wsl-in-vs-code)
+    - [:chart_with_upwards_trend: Real-Time Analytics on Azure Database for PostgreSQL - Hyperscale (Citus)](#-real-time-analytics-on-azure-database-for-postgresql---hyperscale-citus)
 
-## Sentiment Analysis with Python Azure Functions
+### :tada::snake: Welcome to out Python Virtual Labs for PyCon India Online 2020 :tada::snake:
+
+A collection of labs demonstrating how to build Python applications with Azure and [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=pyconindia-github-cxa).
+
+**Complete at least one Python virtual lab to be get an exclusive [Azure Heroes badge](https://www.microsoft.com/skills/azureheroes)!**
+
+### :thinking: How to participate?
+
+1. Join the Microsoft Python Discord (http://aka.ms/python-discord ) and the **#python-virtual-labs** channel to ask questions, get help with labs, and/or further instructions.
+
+2. Select and complete a Python Virtual Lab from this repository (see [Virtual Labs](#-virtual-labs))
+   - :warning: some labs might need an **Azure subscription**. You can sign up for an [Azure free account](https://azure.microsoft.com/free/?WT.mc_id=pyconindia-github-cxa) for lab completion
+
+3. Confirm lab completion and redeem your Azure Hero Badge
+   - You must complete at least one lab to qualify for the giveaway
+   - To demonstrate completion choose one of these options:
+     - Upload a screen capture of your completed labs to our #python-virtual-labs channel
+     - Tweet with the #PyConIndia2020hashtag and tag [@azureadvocates](https://twitter.com/azureadvocates) with a screenshot of your completed lab
+
```

Comment:
@arkodyti I added the instructions here so maybe a guide.md is not necessary? Can you check these are correct?